pvlib/tests/test_irradiance.py

```diff
@@ -295,6 +296,17 @@ def test_liujordan():
     assert_frame_equal(out, expected)
```

Comment:
This test is going to emit a deprecation warning and we want to avoid that. We could delete it or we could skip the new `test_deprecated_09` test, add `@fail_on_pvlib_version('0.9')` around this test, and add the `with pytest.warns...` here

Reply:
I prefer the option after "or"